02-csv-vector-raster-plotting.Rmd

```diff
@@ -115,107 +118,133 @@ head(plot.location)
+If it is not still open in R, let's open up the shapefile `HarClip_UTMZ18".
 
 ```{r read-shp }
-#note: read ogr is preferred as it maintains prj info
+
 aoiBoundary <- readOGR("boundaryFiles/HARV/","HarClip_UTMZ18")
+```
 
-#check the CRS
-crs(aoiBoundary)
+This is the shapefile that we want to match up with our csv file.  It is
+important to first confirm that the CRS of our CSV coordinates is the same as 
+the CRS of the shapefile. We can figure that out by looking at the metadata for
+both files.
 
-#the x,y location values for the dataset are available in 
-#the x and y columns. We can use that for our spatial data point object
-#we know the csv file coordinates are in the same CRS as the other shapefiles.
-#we can assign this to the SpatialPointsDataFrame when we import it.
-plot.locationSp <- SpatialPointsDataFrame(plot.location[,1:2],
-                                                plot.location,
-                                                proj4string = crs(aoiBoundary))
+```{r check-crs-shapefile-csv}
+#check the CRS in shapefile
+crs(aoiBoundary)
+```
 
+Next, we will convert our imported `plot.location` object to a
+`SpatialPointsDataFrame` and assign it a CRS using the shapefile.
+
+The x,y location values for the dataset are available and named "x" (column 1) and
+"y" (column 2).  We can use that for our spatial data point objects. We know the csv file 
+coordinates are in the same CRS as the other shapefiles. We can assign this to the 
+`SpatialPointsDataFrame` when we import it.
+
+``` {r plot-csv-shapefile}
+#
+plot.locationSp <- SpatialPointsDataFrame(plot.location[,1:2],    #x,y coordinates
+                    plot.location,                    #?
+                    proj4string = crs(aoiBoundary))    # assinging the CRS system
+                                                      # in our shapefile
+                                          
 #look at CRS
 crs(plot.locationSp)
 
 ```
 
-Now, we have created a `SpatialPointsDataFrame` object in R using a csv file! 
-The `R` object has a CRS that matches the CRS of our shapefiles. Let's plot our 
-data!
+Now, we have created a `SpatialPointsDataFrame` object in R from a csv file! 
+The `R` object has a CRS that matches the CRS of our shapefiles. 
+
+Let's plot our data!
 
 
 ```{r plot-data }
 
 #plot the points
-#use main = to add a title to the map
+#Add title to map using main=""
 plot(plot.locationSp, main="Map of Study Area")
 
-#let's add the polygon layer to our plot
+#let's add the polygon & tower point layer to our plot
 plot(aoiBoundary, add=TRUE)
+plot (point, col="red", add=TRUE)
 
-#add roads to our plot
-roads <- readOGR("boundaryFiles/HARV/","HARV_roads")
-plot(roads, add=TRUE)
+#add roads to our plot, Lesson00: lines<-readOGR("boundaryFiles/HARV/", "HARV_roads")
+plot(lines, add=TRUE)
 
 ```
 
+What happenned to the tower?  The extent of these data are much larger than the
+bouldery file that we originally originally projected.  
```

Comment:
bouldery?? 

be careful with the word large - i'd say covers a larger area or region